dev-docs/bidders/adot.md

```diff
@@ -3,15 +3,25 @@ layout: bidder
 title: Adot
 description: Prebid Adot Bidder Adapter
 biddercode: adot
-pbjs: true
 media_types: banner, video, native
 gdpr_supported: true
+tcf2_supported: true
+pbjs: true
+pbs: true
 ---
 
+
 ### Table of Contents
```

Comment:
Please remove the table of contents -- this isn't long enough to need it.

Reply:
Thanks for you response, i did the modification.